lib/dradis/plugins/persistant_permissions.rb

```diff
@@ -0,0 +1,42 @@
+module Dradis
+  module Plugins
+    module PersistantPermissions
```

Comment:
What do you think about putting this file under `app/controllers/dradis/plugins/concerns`? I feel it's more appropriate because we're using it in a controller.

Reply:
I do like the idea much better. I tried it, and for some reason it wasn't being loaded. So I moved it here.

I'll move it back.